awx/settings/defaults.py

```diff
@@ -148,7 +148,7 @@ def IS_TESTING(argv=None):
 SITE_ID = 1
 
 # Make this unique, and don't share it with anybody.
-SECRET_KEY = 'p7z7g1ql4%6+(6nlebb6hdk7sd^&fnjpal308%n%+p^_e6vo1y'
+SECRET_KEY = open('/etc/tower/SECRET_KEY', 'rb').read().strip()
```

Comment:
Makes tests fail.  Should we put this into `settings/development.py` instead?

Reply:
We could also wrap this in a `try...except...` and just generate a random value in the except block (so it'll just make a random one for tests)

Reply:
The SECRET_KEY isn't getting mounted into the container at `/etc/tower` anymore for some reason, even though it is being rendered in the docker-compose.yml.

Reply:
I was able to reproduce this locally, it looks like what is happening is that an old tools_awx_1 container was still around and was being used by the docker-compose, so the new volume-mount wasn't being created.  

After removing all containers and re-running `make docker-compose`, the secret_key was properly mounted into the container.

Reply:
I could probably get tests passing by adding a `--force-recreate` flag in the Makefile.

Reply:
Okay, that works too if we can get tests passing.

Reply:
I'll take another look at this. I _really_ don't think we should have a default value for the secret key!

Reply:
@rooftopcellist I don't think `--force-recreate` is going to help here. Can you back out that change please?